### PR TITLE
added export event functionality

### PR DESCRIPTION
**Pull request by jellyyams**

### CODE REVIEW

File: blueprints/public.py [hunk 1]
```diff
@@ -15,7 +19,23 @@ def public_page(page):
...
-    pass
+@public.route('/export/<eventid>', methods=['POST', 'GET'])
+def export_event(eventid):
+    if request.method == 'GET':
+
+        event_data = db.events.find_one({'_id' : ObjectId(eventid)})
+        cal = Calendar()
+        event = Event()
+        event['dtstart'] = convert_time(str(event_data['start']))
+        event['dtend'] = convert_time(str(event_data['end']))
```
Comment (jack-greenberg) [inline blueprints/public.py:30]: Instead of writing your own function, use Python's `datetime` library. Use `datetime.datetime.strptime()`.

File: static/index.js [hunk 2]
```diff
@@ -158,7 +158,24 @@ class App extends React.Component {
...
+            element.style.display = 'none';
+
+            document.body.appendChild(element);
+            //autmoatically downloads ical file
+            element.click();
+            document.body.removeChild(element);
+        })
+        .catch(err => {
+            console.error(err);
+        })
```
Comment (jack-greenberg) [inline static/index.js:178]: yeet!

File: blueprints/public.py [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
+from icalendar import Calendar, Event
 
 public = Blueprint('public', __name__, template_folder='../templates', static_folder='../static/build/')
 
 @public.route('/')
 def public_index():
     return render_template('home.html')
 
-@public.route('/<page>')
+@public.route('/p/<page>')
```
Comment (jack-greenberg) [inline blueprints/public.py:13]: Why did you add `/p/`?

Comment (jellyyams) [reply blueprints/public.py:13]: oops I forgot to get rid of that. I was just making sure that routing worked the way I thought it did
